File: backend-go/internal/modules/system/entity/sys_job.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysJob struct {
	ID             common.Int64String `gorm:"column:id;primaryKey" json:"id"`
	JobName        string             `gorm:"column:job_name" json:"jobName"`
	CronExpression string             `gorm:"column:cron_expression" json:"cronExpression"`
	ExecutorType   string             `gorm:"column:executor_type" json:"executorType"`
	ScriptFile     *string            `gorm:"column:script_file" json:"scriptFile"`
	ScriptArgs     *string            `gorm:"column:script_args" json:"scriptArgs"`
	ClassName      *string            `gorm:"column:class_name" json:"className"`
	MethodName     *string            `gorm:"column:method_name" json:"methodName"`
	Status         int                `gorm:"column:status" json:"status"`
	Remark         *string            `gorm:"column:remark" json:"remark"`
	Deleted        int                `gorm:"column:deleted" json:"deleted"`
	CreatedAt      *time.Time         `gorm:"column:created_at" json:"createdAt"`
	UpdatedAt      *time.Time         `gorm:"column:updated_at" json:"updatedAt"`
}

func (SysJob) TableName() string { return "sys_job" }

File: backend-python/app/modules/system/entity/sys_job.py
from sqlalchemy import BigInteger, Integer, String
from sqlalchemy.orm import Mapped, mapped_column

from app.modules.system.entity.base import Base, TimestampMixin


class SysJob(TimestampMixin, Base):
    __tablename__ = "sys_job"

    id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
    job_name: Mapped[str] = mapped_column(String(64))
    cron_expression: Mapped[str] = mapped_column(String(64))
    executor_type: Mapped[str] = mapped_column(String(32))
    script_file: Mapped[str | None] = mapped_column(String(255), nullable=True)
    script_args: Mapped[str | None] = mapped_column(String(1024), nullable=True)
    status: Mapped[int] = mapped_column(Integer)
    remark: Mapped[str | None] = mapped_column(String(255), nullable=True)
    deleted: Mapped[int] = mapped_column(Integer)
